docs/Team/Working-Remote.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Working Remote"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, the dev team is ~20% full-time remote. Given that, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, and video calls are for anything from client meetings to 1-on-1s to pair programming.
+
+As an organization, we are working to make remote more integrated.
+
+Things we currently do are:
+- ensure company meetings have good audio for online attendees
+- regular 1-on-1 communication with same-department coworkers for pairing on non-client coding projects
+- fly remote employees to the Phoenix office roughly once/quarter for 3-5 days, covering the flight, hotel, and meals (and try to schedule those trips at the same time for the remotees!)
+- send gift cards to remote employees when they miss out on a project team's celebration
+
+Things we want to do are:
+- make Virtual Coffees part of the culture, so an on-site employee can grab a remote coworker, or vice-versa, from any department for a chat every so often
+
+Things we'd like to do:
+- integrate better screensharing and pair-programming tools
+- make the inclusion of a GTM or Google Meet the default when creating meetings
+- have a shared microphone in larger company meetings that can be passed around, or have a culture where questions that are asked from the room are repeated into the central mic for the benefit of remote attendees
```

Comment:
I dunno about this one... The majority of our meetings don't have a ton of people in one room. It's only for all-hands that this is an issue (I think?). The A/V issues also get complex... I'd file this as a nice-to-have but realistically wouldn't make a move on it for a while longer.